packages/container/lib/container.js

```diff
@@ -1,8 +1,19 @@
-import { dictionary, symbol, setOwner, OWNER, NAME_KEY } from 'ember-utils';
+/* globals Proxy */
+import {
+  dictionary,
+  symbol,
+  setOwner,
+  OWNER,
+  assign,
+  NAME_KEY
+} from 'ember-utils';
 import { ENV } from 'ember-environment';
-import { assert, deprecate, runInDebug } from 'ember-metal';
+import { assert, deprecate, runInDebug, isFeatureEnabled } from 'ember-metal';
 
 const CONTAINER_OVERRIDE = symbol('CONTAINER_OVERRIDE');
+const HAS_PROXY = typeof Proxy === 'function';
```

Comment:
Can we move this into `ember-utils` and export as `HAS_NATIVE_PROXY`?  Very similar to what was done in https://github.com/emberjs/ember.js/pull/14649 for `packages/ember-utils/lib/weak-map-utils.js`.